scala/scala_maven_import_external.bzl

```diff
@@ -187,19 +187,34 @@ def scala_maven_import_external(
     artifact,
     server_urls,
     rule_load = "load(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")",
+    src_artifact = None,
     **kwargs):
   jvm_maven_import_external(
       rule_name = "scala_import",
       rule_load = rule_load,
       artifact = artifact,
       server_urls = server_urls,
+      src_artifact = src_artifact,
       #additional string attributes' values have to be escaped in order to accomodate non-string types
       #    additional_rule_attrs = {"foo": "'bar'"},
       **kwargs)
 
-def jvm_maven_import_external(artifact, server_urls, **kwargs):
+def jvm_maven_import_external(artifact,
+                              server_urls,
+                              src_artifact = None,
+                              **kwargs):
+  if kwargs.get("srcjar_urls") and src_artifact:
+    fail("Either use srcjar_urls or src_artifcat but not both")
```

Comment:
Artifact is misspelled. Would you be interested in a fixup PR for this message?

Reply:
Sorry for the oversight :( @anchlovi can you take care of it?

Reply:
https://github.com/bazelbuild/rules_scala/pull/624